src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
```

Comment:
let's do it in a loop instead

Reply:
it`s look like the
Lottery lottery = new Lottery();
for (int i = 0; i < 3; i++) {
    System.out.println(lottery.getRandomBall());
}